docs/reference/schema.md

```diff
@@ -85,6 +85,14 @@ Allows configuring the Docker network used for `tye run`.
 
 If a network is configured, then all services running in containers will connect to the specified network. Otherwise a Docker network will be created with a generated name, and used to connect all containers.
 
+#### `dashboardPort` (int)
```

Comment:
Would it be possible to add this same information to the [YAML (i.e. JSON) schema](https://github.com/dotnet/tye/blob/main/src/schema/tye-schema.json) for configuration file.  That schema is used by IDEs like VS Code to provide IntelliSense to users as they edit the file.

Reply:
Done!